db/rdbms/src/main/java/io/camunda/db/rdbms/write/service/RdbmsPurger.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Camunda License 1.0. You may not use this file
+ * except in compliance with the Camunda License 1.0.
+ */
+package io.camunda.db.rdbms.write.service;
+
+import io.camunda.db.rdbms.sql.PurgeMapper;
+import java.util.List;
+
+public class RdbmsPurger {
+
+  private static final List<String> TABLE_NAMES =
+      List.of(
+          "EXPORTER_POSITION",
+          "PROCESS_DEFINITION",
+          "PROCESS_INSTANCE",
+          "VARIABLE",
+          "FLOW_NODE_INSTANCE",
+          "USER_TASK",
```

Comment:
Since we don't have foreign keys, it doesn't matter, but for me it feels uncomfortable to truncate first the main table and then the children tables. :D Can we first truncate Candidate stuff, and then user_task? :D (Same for others)